pkg/sysregistries/system_registries.go

```diff
@@ -30,7 +31,17 @@ type tomlConfig struct {
 // Reads the global registry file from the filesystem. Returns
 // a byte array
 func readRegistryConf() ([]byte, error) {
-	configBytes, err := ioutil.ReadFile(systemRegistriesConfPath)
+	var configFilePath string
+	// If an environment variable for the configuration file
+	// exists, use it over the the default location.  This is
+	// useful for testing purposes in cri-o.
+	altConfPath := os.Getenv("REGISTRIES_CONF_PATH")
```

Comment:
If this is _only_ for testing, it seems overkill to have it as an user-accessible environment variable in all installations.

Would it be possible to instead use `types.SystemContext`, e.g. the way `docker/lookaside.go:registriesDirPath` works, and for the tests to use an appropriately initialized `types.SystemContext`?

Reply:
I'll look into it ...